native_client/deepspeech.h

```diff
@@ -30,8 +30,8 @@ typedef struct MetadataItem {
 typedef struct Metadata {
   MetadataItem* items;
   int num_items;
-  // Approximated probability (confidence value) for this transcription.
-  double probability;
+  // Approximated confidence value for this transcription.
```

Comment:
Given the issue that leads to that change, maybe we should elaborate a bit more on what this is exactly, how to interpret values, etc. ?